features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -2361,8 +2361,20 @@ CREATE TABLE IF NOT EXISTS AM_CORRELATION_PROPERTIES(
     FOREIGN KEY (COMPONENT_NAME) REFERENCES AM_CORRELATION_CONFIGS(COMPONENT_NAME) ON DELETE CASCADE
 );
 
+  CREATE TABLE IF NOT EXISTS AM_INTERNAL_TOKEN_REVOCATION_CONSUMER_KEY_EVENTS (
```

Comment:
We will need to add these tables to other DB scripts as well.

Reply:
Fixed by. https://github.com/wso2/carbon-apimgt/pull/12174/commits/70b076a6067da5daa54577ca7065398e9a9c5e7d